src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -284,6 +286,7 @@ def __del__(self) -> None:
             self._method,
             serialized_request,
             self._deadline,
+            self._metadata,
```

Comment:
From what I can undestand here we would need to provide yes or yes a tuple, so a `None` object won't be valid since the signature for the function says that a `tuple` parameter is expeted.

am I missing something ? or is there a place where the `None` default value is convderted to an empty tuple?

Reply:
You are right, I need to convert the `None` value to empty tuple.